engine/src/main/java/org/destinationsol/assets/json/Validator.java

```diff
@@ -26,18 +27,24 @@
 
     static Logger logger = LoggerFactory.getLogger(Validator.class);
 
-    public static void validate(JSONObject json, String schemaPath) {
+    public static void validate(String jsonPath, String schemaPath) {
+        JSONObject json = Assets.getJson(jsonPath).getJsonValue();
         JSONObject schema;
 
         try {
             schema = Assets.getJson(schemaPath).getJsonValue();
         } catch (RuntimeException e) {
-            logger.warn("Json Schema " + schemaPath + " not found!", e);
+            logger.warn("Json Schema " + schemaPath + " not found!");
```

Comment:
You've given a very specific warning message to a rather ambigous exception. The schema file being present, but *invalid*, also causes this. Please clear that up :)

Reply:
Actually, I was the one to have added org.json and schema validation, so yes, that warning code was indeed written by me. I'll fix it soon, thank you for bringing it to my notice.

Reply:
The code now checks if the RTE is for file not found. Hopefully this is an appropriate solution.